### DISCUSSION TITLE
🚀 Latest models

### DISCUSSION THREAD

**Discussion by chiefMarlin** [oobabooga/text-generation-webui#4336] [General]
Can we have an ongoing poll of the best latest models available ?
Here are a few to get started:

`TheBloke/Mistral-7B-Instruct-v0.1-GGUF`
`TheBloke/CodeLlama-13B-GPTQ`

**Comment by viperwasp** [comment]
It seems like this is a terrible place for this kind of community. If anyone has a better forum with active people please let us know.

**Comment by FartyPants** [comment]
The best place to discuss this is in reddit locallama 
(I don't mean it in wrong way, I truly believe it)
